typescript/cli/src/deploy/utils.ts

```diff
@@ -55,6 +56,31 @@ export async function runPreflightChecksForChains({
   if (sufficient) logGreen('✅ Balances are sufficient');
 }
 
+export async function runDeployPlanStep({
+  context,
+  chain,
+}: {
+  context: WriteCommandContext;
+  chain: ChainName;
+}) {
+  const { signer, skipConfirmation } = context;
+  const address = await signer.getAddress();
+
+  logBlue('\nDeployment plan');
+  logGray('===============');
+  log(`Transaction signer and owner of new contracts will be ${address}`);
```

Comment:
is this always true or is this an assumption?

Reply:
It's generally true. The deployer will be the tx signer and owner of most contracts. proxy contracts will be the proxy admin. But the owner of proxy admin will be the deployer